feature/gribi/ate_tests/hierarchical_weight_resolution_test/README.md

```diff
@@ -86,28 +84,29 @@ WCMP width of 16 nexthops:
 
     *   IPv4Entry 192.0.2.222/32 in default VRF, pointing to NextHopGroup(NHG#3)
         in default VRF, with 16 NextHops(NH#100, NH#101, ..., NH#115), all with
-        weight: 1, in default VRF:
+        weight: 16 except NHG#100 is of weight 1, in default VRF:
 
         *   NH#100 with weight:1, pointing to 192.0.2.18
 
-        *   NH#101 with weight:1, pointing to 192.0.2.22
+        *   NH#101 with weight:16, pointing to 192.0.2.22
 
         *   ...
 
-        *   NH#115 with weight:1, pointing to 192.0.2.79
+        *   NH#115 with weight:16, pointing to 192.0.2.79
 
 *   Validate with traffic:
 
     *   NH10: (1/32) * (3/8) = 1.171% traffic received by ATE port-2 VLAN 1
```

Comment:
my PR #1133 says `~`, but here is `=`, any reason why?

Reply:
done